service/src/test/resources/core-case-data-store-api/callback-request.json

```diff
@@ -495,7 +495,8 @@
         "title": "Example Order",
         "details": "Example order details here - Lorem ipsum dolor sit amet, consectetur adipiscing elit"
       },
-      "familyManCaseNumber": "12345L"
+      "familyManCaseNumber": "12345L",
+      "representatives": []
```

Comment:
Just keeping a house a home here? Sorry, I did not see where this is being used.

Reply:
removed